shell/common/node_bindings.cc

```diff
@@ -478,7 +478,7 @@ node::Environment* NodeBindings::CreateEnvironment(
   } else {
     // Match Blink's behavior by allowing microtasks invocation to be controlled
     // by MicrotasksScope objects.
-    is.policy = v8::MicrotasksPolicy::kScoped;
+    is.policy = context->GetIsolate()->GetMicrotasksPolicy();
```

Comment:
This comment is now incorrect, right? Is the new behavior safe? Why was this the way it was before?

Reply:
It was introduced in 103fea5cdfe3a3fec62d82f91b20eda33cad820d originally by @zcbenz . @zcbenz could you take a look at this PR?

I believe it is safe to do it the way because everything outside of `UvRunOnce()` would have scoped policy, and all C++ code invokes from `UvRunOnce()` will have `kExplicit`. This PR doesn't change this behavior at all, but just makes sure that we don't change back to `kScoped` while we are in `UvRunOnce()`.

As stated in the PR description, using `kExplicit` with `MicrotasksScope` is safe because of [this check in v8](https://github.com/v8/v8/blob/5f8edcb1be1718e9648aa1250bf5faf6d6ef9546/src/api/api.cc#L9219-L9225).

Reply:
Changed the comment to reflect new behavior.

Reply:
> It was introduced in [103fea5](https://github.com/electron/electron/commit/103fea5cdfe3a3fec62d82f91b20eda33cad820d) originally by @zcbenz . @zcbenz could you take a look at this PR?

I don't quite remember the background of the change, but we have a test for the original issue so as long as the tests are passing there should be no problem.

Reply:
Fixed.